### PR TITLE
6.0.1

### PR DESCRIPTION
**Pull request by rekmarks**
- `6.0.1`
- Update changelog
  - #68 
  - #69

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.1] - 2020-07-15
+
+### Added
+
+- Warning message for the `data` event
```
Comment (Gudahtt) [inline CHANGELOG.md:14]: I would categorize this as `fixed`; it had a pre-existing warning message of `undefined`. Less than helpful, but it existed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - This deprecated event was added back in `6.0.0`, but the warning message was not defined.
+
+### Changed
+
+- Restore `publicConfigStore` property as alias for `_publicConfigStore`
```
Comment (Gudahtt) [inline CHANGELOG.md:16]: Nit: I would categorize this as `fixed` as well. Renaming the property may have been intentional, but presumably the dapp-breakage it resulted in was not, so I think it's fair to characterize it as a bug.

Comment (rekmarks) [reply CHANGELOG.md:16]: Hah, totally forgot/missed that `Fixed` was a thing in KeepAChangeLog. Moved both to that category and eliminated `Added`/`Changed`.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**